@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -458,6 +411,85 @@ export class PoiManager {
         }
     }
 
+    /**
+     * Create and add POI [[TextElement]]s to tile with a series of positions.
+     */
+    private addPoi(
+        tile: Tile,
+        poiGeometry: PoiGeometry,
+        technique: PoiTechnique,
+        techniqueIdx: number,
+        positions: THREE.BufferAttribute
+    ) {
+        const poiTechnique = technique as PoiTechnique;
+        let imageTextureName = poiTechnique.imageTexture;
+        if (typeof poiTechnique.imageTexturePrefix === "string") {
+            imageTextureName = poiTechnique.imageTexturePrefix + imageTextureName;
```

Comment:
My opinion is that this duplicating "add prefix/postfix" code should be separated in a function. Something like `getImageTextureName(technique)`.

Reply:
Will try something.